rustfmt.toml

```diff
@@ -1,7 +1 @@
-array_layout = "Block"
-generics_indent = "Block"
-fn_args_layout = "Block"
-fn_call_style = "Block"
-max_width = 115
-where_style = "Rfc"
-write_mode = "overwrite"
+hard_tabs = true
```

Comment:
Nope, not in this crate, sorry. Please instead remove the `rustfmt.toml` file and re-do the `cargo fmt`. Please also rebase the fix into this commit so that the entire crate isn't touched by this commit.

Reply:
If you'd prefer, I can make the 2018-ification / formatting changes myself in a separate PR, and then you can rebase afterwards.

Reply:
Decided that it was quick enough to do that, so I've opened #2.

Reply:
And merged. Please rebase on master.